presentations/en/2021-micro-acquisition-elevator-pitch.md

```diff
@@ -0,0 +1,103 @@
+<!--markdownlint-disable MD003 MD033 MD041 MD036 -->
+
+<style>
+.reveal section img { background:none; border:none; box-shadow:none; }
+</style>
+
+# Micro-Acquisition Pilot
+
+ESDC IT Strategy Team
+
+Note:
+
+- we are here today to talk to you about our procurement pilot and we will
+- but first we want to talk a little about what we believe
+
+---
+
+Trying to continuously improve while also keeping the lights on is hard
+
+Note:
+
```

Comment:
Keep things small, break down big problems in smaller, self contained ones. Reuse, reuse, reuse. Share, share, share.

Reply:
Not sure it goes here btw